webui/package.json

```diff
@@ -25,10 +25,12 @@
     "@types/react-virtualized": "^9.22.0",
     "antd": "^5.22.2",
     "buffer": "^6.0.3",
+    "cron-schedule": "^5.0.4",
     "cross-env": "^7.0.3",
     "events": "^3.3.0",
     "lodash": "^4.17.21",
     "parcel": "^2.13.2",
+    "pretty-ms": "^9.2.0",
```

Comment:
We can avoid this dependency -- there's a `formatDuration` utility in the webui already that takes a time in milliseconds https://github.com/garethgeorge/backrest/blob/fc4bdc089ad456defc38f52b00ff3c5c2a566fda/webui/src/lib/formatting.ts#L63